### ISSUE TITLE
background-image: image-set is not being prefixed

### ISSUE THREAD

**Issue by G07cha** [thysultan/stylis.js#105]
With latest version:

```javascript
var stylis = require("stylis")

stylis('#id', `
* {
background-image: image-set(url(foo.jpg) 2x);
}
`)
```

Returns:

```
"#id *{background-image:image-set(url(foo.jpg) 2x);}"
```

Although for Chrome it should be prefixed like so:

```
"#id *{background-image:-webkit-image-set(url(foo.jpg) 2x);}"
```

**Comment by thysultan** [comment]
Patched in 3.5.1

**Comment by 1000hz** [comment]
This should also be prefixed for `cursor`
